src/interfaces/wallet.h

```diff
@@ -5,10 +5,11 @@
 #ifndef BITCOIN_INTERFACES_WALLET_H
 #define BITCOIN_INTERFACES_WALLET_H
 
+#include <addresstype.h>
 #include <consensus/amount.h>
 #include <interfaces/chain.h>          // For ChainClient
 #include <pubkey.h>                    // For CKeyID and CScriptID (definitions needed in CTxDestination instantiation)
```

Comment:
Maybe fixup the comment on pubkey.h here too

Reply:
We can just drop any `// For ...` comments entirely

Reply:
Dropped those comments.